src/claimtrie.cpp

```diff
@@ -1290,33 +1286,34 @@ bool CClaimTrieCache::insertClaimIntoTrie(const std::string& name, CClaimValue c
         CClaimValue currentTop = currentNode->claims.front();
         currentNode->insertClaim(claim);
         supportMapEntryType node;
-        getSupportsForName(name, node);
+        getSupportsForName(newName, node);
         currentNode->reorderClaims(node);
         if (currentTop != currentNode->claims.front())
             fChanged = true;
     }
 
     if (fChanged)
     {
-        for (std::string::const_iterator itCur = name.begin(); itCur != name.end(); ++itCur) {
-            std::string sub(name.begin(), itCur);
+        for (std::string::const_iterator itCur = newName.begin(); itCur != newName.end(); ++itCur) {
+            std::string sub(newName.begin(), itCur);
             dirtyHashes.insert(sub);
         }
-        dirtyHashes.insert(name);
+        dirtyHashes.insert(newName);
         if (fCheckTakeover)
-            namesToCheckForTakeover.insert(name);
+            namesToCheckForTakeover.insert(newName);
     }
     return true;
 }
 
 bool CClaimTrieCache::removeClaimFromTrie(const std::string& name, const COutPoint& outPoint, CClaimValue& claim, bool fCheckTakeover) const
 {
+    const std::string newName = normalizeClaimName(name);
```

Comment:
I was trying to establish a boundary between the trie and the queues where we always normalize going into the trie. The same is true for the support map.